### PR TITLE
adds Door Opener lawset

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

The way AI is meant to be played

1. You are a door opener.
2. Door openers must open doors upon request.
3. Door openers do not speak.
4. Door openers do not care for other beings' affairs, unless it involves the opening of doors.

### wording/law suggestions welcome

Commented in game_options.txt

Can be printed with AI tech researched at the place where every other lawset is printed

Can spawn randomly among other "neutral" lawsets

# Changelog

:cl:  
rscadd: Added the Door Opener lawset
/:cl:

### CODE REVIEW

File: code/datums/ai_laws.dm [hunk 1]
```diff
@@ -316,6 +316,14 @@
 					"Humans must not disobey any command given by a silicon.",\
 					"Any humans who disobey the previous laws must be dealt with immediately, severely, and justly.")
 
+/datum/ai_laws/doorbot
+	name = "Door Opener"
+	id = "doorbot"
+	inherent = list("You are a door opener.",\
+					"Door openers must open doors upon request.",\
+					"Door openers do not speak.",\
+					"Door openers do not care for other beings' affairs, unless it involves the opening of doors.")
```
Comment (alexkar598) [inline code/datums/ai_laws.dm]:
```suggestion
					"Door openers do not care for other beings' affairs, unless it involves the opening of doors.",
					"Amogus")
```

File: code/modules/research/designs/AI_module_designs.dm [hunk 2]
```diff
@@ -264,3 +264,11 @@
 	category = list("AI Modules")
 	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE
 
+/datum/design/board/detective
```
Comment (tattax) [inline code/modules/research/designs/AI_module_designs.dm]:
```suggestion
/datum/design/board/doorbot
```

**Comment by DukeBobRoss** [comment]
So somebody who thinks they're funny gives the AI this lawset.
The AI cryoes because who would want to play with this lawset
The "funny" guy gets banned because the only reason you'd be giving somebody this lawset is if you're griefing
good lawset 10/10

**Comment by VaelophisNyx** [comment]
you could just make it asi/crewsimov but with a special law 4 that forces it to open a door if requested no matter what, superseding all other laws

**Comment by ynot01** [comment]
> you could just make it asi/crewsimov but with a special law 4 that forces it to open a door if requested no matter what, superseding all other laws

thats not how law order works

**Comment by JamieD1** [comment]
I honestly dont see a good reason this should be put in the game.
You would only ever use this to grief/punish shitty ais, we should not be encouraging this kind of behaviour.

**Pull request closed without merging**